ffi/examples/visit-expression/CMakeLists.txt

```diff
@@ -8,26 +8,16 @@ target_link_directories(visit_expression PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../
 target_link_libraries(visit_expression PUBLIC delta_kernel_ffi)
 target_compile_options(visit_expression PUBLIC)
 
-target_compile_options(visit_expression PRIVATE -Wall -Wextra -Wpedantic -Werror -Wno-strict-prototypes)
-
-# Get info on the OS and platform
-if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-    set(MACOSX TRUE)
-endif()
-if(UNIX AND NOT APPLE)
-    set(LINUX TRUE)
+if(MSVC)
+  target_compile_options(read_table PRIVATE /W4 /WX)
+else()
+  # no-strict-prototypes because arrow headers have fn defs without prototypes
+  target_compile_options(visit_expression PRIVATE -Wall -Wextra -Wpedantic -Werror -Wno-strict-prototypes -g -fsanitize=address)
+  target_link_options(visit_expression PRIVATE -g -fsanitize=address)
```

Comment:
do we want to keep the linux-specific check so we can do fsanitize=leak only on linux? PR desc said that Darwin doesn't support it right?

Reply:
Turns out that `-fsanitize=address` includes leak detection, if supported. I had to write C code to disable it. 
Asking for `-fsanitize=leak` specifically will fail on darwin because not supported.

Reply:
(after leaks are fixed)